ui/job-view/details/DetailsPanel.jsx

```diff
@@ -1,16 +1,12 @@
-/* eslint import/no-unresolved: [2, { ignore: ['@mozilla/glean/web$'] }] */
-
 import React from 'react';
 import PropTypes from 'prop-types';
 import chunk from 'lodash/chunk';
 import { connect } from 'react-redux';
-import Glean from '@mozilla/glean/web';
 
 import { notify } from '../redux/stores/notifications';
```

Comment:
Is this still needed? The export at the end of the file also got added when it got added with the Glean support.

Reply:
thanks for finding this, I went through the original commit and found a few other areas that needed cleaning up.